lineman/app/components/thread_page/proposal_collapsed/proposal_collapsed.haml

```diff
@@ -1,9 +1,9 @@
 %a.proposal-collapsed.media{href: '', id: 'proposal-{{proposal.key}}'}
   .sr-only
     {{proposal.name}}.
-    %closing_in{time: 'proposal.closingAt'}.
+    %proposal_closing_time{proposal: 'proposal'}.
   .screen-only{aria-hidden: 'true'}
-    %closing_in.pull-right{time: 'proposal.closingAt'}
+    %proposal_closing_time.pull-right{proposal: 'proposal'}.
```

Comment:
Sorry, this is the ultimate nit-pick, but I'm just wondering what that dot is there for?